builds/checkin/api-proxy.yaml

```diff
@@ -0,0 +1,81 @@
+trigger: none
+pr:
+  branches:
+    include:
+      - master
```

Comment:
why change?

Reply:
One of Bear comments.

Reply:
it is ok to start with master.  But you need to wait for this commit merging to other branch, then enable it for that branch here.  Please remember to follow it in iiot branch as if you don't include iiot branch, api-proxy module may be broken in iiot.

Reply:
Yes Bear said the same thing.